Tests/Functional/Controller/ImportTest.php

```diff
@@ -71,26 +56,41 @@ public function test_assign_remote_to_local_category_action_without_params()
         $this->assertFalse($responseData['success']);
     }
 
-    public function test_assign_remote_to_local_category_action()
+    public function test_assign_remote_to_local_category()
     {
         /** @var TestClient $client */
-        $client = $this->createBackendClient();
+        $client = self::createBackendClient();
+
+        $connection = self::getDbalConnection();
 
         $this->importFixturesFileOnce(__DIR__ . '/../_fixtures/categories.sql');
 
+        $localCategoryId = '140809703';
+        $remoteCategoryKey = '/deutsch/television';
+
+        $remoteCategoryId = $connection->fetchColumn('SELECT id FROM s_plugin_connect_categories WHERE category_key = ?', [$remoteCategoryKey]);
```

Comment:
IDK whats in the fixture but i think that there's the shopID 1234 for that remote category
so please use it in this query too because the category key alone isn't unique anymore